src/App.js

```diff
@@ -1,13 +1,28 @@
 import React from "react";
-
 import Bookings from "./Bookings";
+import SearchResults from "./SearchResults";
+import Footer from "./Footer";
+import TouristInfoCards from "./TouristInfoCards";
+import Heading  from "./Heading";
+import Restaurant from "./Restaurant";
 import "./App.css";
 
+// what is key about...
+
+
 const App = () => {
+  const contact = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789",
+  ];
```

Comment:
It could better to define this variable outside of the app component. Like this we are recreating it every time the App component rerenders and we dont need to 😄